packages/accordions/src/index.ts

```diff
@@ -6,7 +6,10 @@
  */
 
 export { Stepper } from './elements/stepper/Stepper';
+export type { IStepperProps } from './elements/stepper/Stepper';
```

Comment:
There were a number of missing prop interface exports. This PR recovered many (probably not all) so they could be reused within `Story` modules.